packages/apps/dashboard/ui-2024/.prettierrc

```diff
@@ -3,6 +3,6 @@
 	"semi": true,
```

Comment:
indent changed to spaces, we need to apply it to other files in this package. I didn't do it to show only related changes to CR fixes